specification/src/main/asciidoc/platform/ApplicationProgrammingInterface.adoc

```diff
@@ -311,6 +304,52 @@ an exception to this in order to accommodate interface type dependencies—for e
 the Jakarta™ Enterprise Beans SessionContext dependency on the
 _javax.xml.rpc.handler.MessageContext_ type.]
 
+The following specifications are outside Jakarta EE, provided by the Java™ runtime.
```

Comment:
If we're not sure of the Y's and N's for this chart, then maybe just leave this table out and we'll go with all of the rest of the changes?

Reply:
If that table brings hesitation, let's delete it and revisit in a future specification version.

Reply:
If we were just moving items from the original table to this new one, I would be okay with it.  But, I'm just not positive on the applicability of these technologies to all three containers, especially the AppClient.  I'll update the PR.  Thanks.